### PR TITLE
support for ccc refunds and reference field change

### PR DESCRIPTION
**Pull request by harshithaputtaswamy**

### CODE REVIEW

File: apps/xero/models.py [hunk 1]
```diff
@@ -85,6 +85,7 @@ def get_item_code_or_none(expense_group: ExpenseGroup, lineitem: Expense):
         ).first()
 
         if mapping:
+            print(mapping)
```
Comment (Shwetabhk) [inline apps/xero/models.py:88]: remove print statement

File: apps/xero/models.py [hunk 2]
```diff
@@ -177,7 +178,7 @@ def create_bill(expense_group: ExpenseGroup):
             defaults={
                 'currency': expense.currency,
                 'contact_id': contact.destination.destination_id,
-                'reference': '{} - {}'.format(expense_group.id, expense.employee_email),
+                'reference': expense.report_id,
```
Comment (Shwetabhk) [inline apps/xero/models.py:181]:
```suggestion
                'reference': expense.claim_number,
```

File: apps/xero/utils.py [hunk 3]
```diff
@@ -13,6 +13,7 @@
 from xerosdk import XeroSDK
 from xerosdk.exceptions import WrongParamsError
 
+from fyle_xero_api import settings
```
Comment (Shwetabhk) [inline apps/xero/utils.py:16]: where are we using this?

Comment (harshithaputtaswamy) [reply apps/xero/utils.py:16]: For report and expense url

File: apps/xero/utils.py [hunk 4]
```diff
@@ -460,14 +461,17 @@ def __construct_bill(self, bill: Bill, bill_lineitems: List[BillLineItem]) -> Di
         """
         general_mappings = GeneralMapping.objects.filter(workspace_id=self.workspace_id).first()
         general_settings = WorkspaceGeneralSettings.objects.get(workspace_id=self.workspace_id)
+        workspace = Workspace.objects.get(id=self.workspace_id)
```
Comment (Shwetabhk) [inline apps/xero/utils.py:464]:
```suggestion
        workspace = bill.expense_group.workspace
```

File: apps/xero/utils.py [hunk 5]
```diff
@@ -546,6 +550,7 @@ def __construct_bank_transaction(self, bank_transaction: BankTransaction,
...
```
Comment (Shwetabhk) [inline apps/xero/utils.py:553]:
```suggestion
        workspace = bank_transaction.expense_group.workspace
```

File: docker-compose-pipeline.yml [hunk 6]
```diff
@@ -26,6 +24,7 @@ services:
       FYLE_CLIENT_ID: ''
       FYLE_CLIENT_SECRET: ''
       FYLE_REFRESH_TOKEN: ''
+      FYLE_APP_URL: 'https://app.fyle.tech'
```
Comment (Shwetabhk) [inline docker-compose-pipeline.yml:27]: let's not hardcode please

File: docker-compose.yml.template [hunk 7]
```diff
@@ -22,6 +22,7 @@ services:
       FYLE_CLIENT_ID:
       FYLE_CLIENT_SECRET:
       FYLE_TOKEN_URI:
+      FYLE_APP_URL: 'https://app.fyle.tech'
```
Comment (Shwetabhk) [inline docker-compose.yml.template:25]: lets not hardcode

File: docker-compose.yml.template [hunk 8]
```diff
@@ -49,6 +50,7 @@ services:
...
```
Comment (Shwetabhk) [inline docker-compose.yml.template:53]: lets not hardcode

**Review by Shwetabhk — APPROVED** [review]

**Review by Shwetabhk — CHANGES_REQUESTED** [review]
Please fix comments

**Review by Shwetabhk — APPROVED** [review]

**Pull request merged**